src/test/features/response/tasks/yourRepaymentPlanTask.ts

```diff
@@ -0,0 +1,79 @@
+/* tslint:disable:no-unused-expression */
+import { expect } from 'chai'
+
+import { FullAdmission, ResponseDraft } from 'response/draft/responseDraft'
+import { DefendantPaymentOption, DefendantPaymentType } from 'response/form/models/defendantPaymentOption'
+import { PayBySetDate as PaymentDate } from 'forms/models/payBySetDate'
+import { MomentFactory } from 'shared/momentFactory'
+import { DefendantPaymentPlan as PaymentPlan } from 'response/form/models/defendantPaymentPlan'
+import { PaymentSchedule } from 'ccj/form/models/paymentSchedule'
+import { localDateFrom } from 'test/localDateUtils'
+import { IndividualDetails } from 'forms/models/individualDetails'
+import { Defendant } from 'drafts/models/defendant'
+import { ResponseType } from 'response/form/models/responseType'
+import { Response } from 'response/form/models/response'
+import { YourRepaymentPlanTask } from 'features/response/tasks/yourRepaymentPlanTask'
+import { PaymentType } from 'features/ccj/form/models/ccjPaymentOption'
+
+function validResponseDraftWith (paymentType: DefendantPaymentType): ResponseDraft {
```

Comment:
Please remove `case DefendantPaymentType.BY_SET_DATE` from the method. Also no point having `paymentType: DefendantPaymentType` as argument since that task only operates on payments by instalments.

Reply:
👍